examples/image-snapshot/test/basic.test.ts

```diff
@@ -0,0 +1,17 @@
+import { readFileSync } from 'fs'
+import { expect, test } from 'vitest'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+
+declare global {
+  namespace jest {
+    interface Matchers<R> {
+      toMatchImageSnapshot(): R
+    }
+  }
+}
+
+expect.extend({ toMatchImageSnapshot })
+
+test('image snapshot', () => {
+  expect(readFileSync('./test/stubs/input-image.png')).toMatchImageSnapshot()
```

Comment:
Do we have a license/credit for this image?

Reply:
Updated it with the vitest logo, so we don't have to get into those concerns (Just figured I'll use a real example from my code ;) ).